### ISSUE TITLE
Feature Request: Blackout screens shortcut

### ISSUE THREAD

**Issue by delta25** [lhc70000/iina#1215]
Running:
- macOS 10.13
- IINA 0.0.14.1

Blacking out screens when going to fullscreen is fantastic, but having a shortcut to toggle it on and off - perhaps your email dinged and you want to check on monitor A without going out of fullscreen on monitor B - would be great.

**Comment by LEMONed** [comment]
I agree. A keyboard shortcut and/or a more accessible toggle on the quick settings panel would be nice.
